src/main/steps/common/task-list/utils.ts

```diff
@@ -104,3 +105,11 @@ export const isCafcassServed = (caseData: Partial<CaseWithId>): boolean => caseD
 
 export const isCafcassCymruServed = (caseData: Partial<CaseWithId>): boolean =>
   caseData?.isCafcassCymruServed === YesOrNo.YES;
+
+export const hasResponseBeenSubmitted = (caseData: Partial<CaseWithId>, respondent: Respondent): boolean => {
+  return !!caseData.citizenDocuments?.find(
```

Comment:
check for length as well caseData.citizenDocuments && caseData.citizenDocuments.length  && caseData.citizenDocuments?.find